spond/spond.py

```diff
@@ -1,74 +1,32 @@
 #!/usr/bin/env python3
-
 from datetime import datetime
 from typing import TYPE_CHECKING, List, Optional
 
-import aiohttp
+from .base import SpondBase
 
 if TYPE_CHECKING:
     from datetime import datetime
 
 
-class AuthenticationError(Exception):
-    pass
-
-
-class Spond:
+class Spond(SpondBase):
 
-    API_BASE_URL = "https://api.spond.com/core/v1/"
```

Comment:
Dropped my API_BASE_URL class constant refactor here

Reply:
I already did that here: https://github.com/Olen/Spond/commit/9391ef278578e7b6b50ab0e7463a401162c751e8

You are including it in both `__super__()` calls so it should (remain) a base class constant.

(Apologies for any terseness, don't have much time today)

Reply:
Ok got it, I think i know what you mean, i won't get to it until today's evening just FYI

Reply:
Ok so spond API url is https://api.spond.com/core/v1/ and the club API is https://api.spond.com/club/v1/ 

They're different so i don't think it should be a base class constant?

Reply:
or did i miss your point completely?

Reply:
Oops, sorry, misread!